lib/cookie.js

```diff
@@ -619,11 +619,11 @@ function parse(str, options) {
           case "lax":
             c.sameSite = "lax";
             break;
+          case "none":
+            c.sameSite = "none";
+            break;
           default:
-            // RFC6265bis-02 S5.3.7 step 1:
-            // "If cookie-av's attribute-value is not a case-insensitive match
-            //  for "Strict" or "Lax", ignore the "cookie-av"."
-            // This effectively sets it to 'none' from the prototype.
+            c.sameSite = undefined;
```

Comment:
How does behavior change with `sameSite` explicitly set to `undefined` vs unset (implicitly `undefined`)?

Reply:
it would catch cases where the SameSite attribute is defined but the value is invalid.  e.g.; `a=0; SameSite=bad`

Reply:
@wjhsf -- can you please let me know if you think we can resolve this issue?